.circleci/config.yml

```diff
@@ -0,0 +1,127 @@
+# These environment variables must be set in CircleCI UI
+#
+# DOCKERHUB_REPO - docker hub repo, format: <username>/<repo>
+# DOCKER_USER - Login user for docker hub
+# DOCKER_PASS - Login password for docker hub user
+version: 2.0
+
+jobs:
+  build: # build for the master branch and tags
+    machine: true
+    working_directory: ~/addons-frontend
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: sudo apt-get update; sudo apt-get install pigz
```

Comment:
Does parallel gzip really improve performance here so that installing that dependency, querying all package sources would still be faster? Querying and installing external dependencies always adds another layer of potential intermittent failures so if we don't need them we shouldn't use them